DokuwikiXMLExport.php

```diff
@@ -79,17 +79,17 @@ private function getPageIds()
         $indexer = new Doku_Indexer();
         $pagesAndDeletedPages = $indexer->getPages();
 
-        $excludedPages = $this->splitConfigToArray($this->conf['plugin']['findologicxmlexport']['excludePages']);
-
-        foreach ($pagesAndDeletedPages as $page) {
-            if (p_get_metadata($page)['description']) { // Only get pages with content
-                if (!in_array($page, $excludedPages)) { // Exclude pages from config
-                    $ids[] = $page;
-                }
+        // Get all pages that do have a description
+        $pagesAndDeletedPages = array_filter($pagesAndDeletedPages, function ($page, $k) {
+            if (p_get_metadata($page)['description']) {
```

Comment:
You can make it even shorter by just using the expression as a return value, something like `return (p_get_metadata($page)['description'] !== '');`

Reply:
Yep.